pkg/packagekit/updates.jsx

```diff
@@ -995,7 +947,7 @@ class OsUpdates extends React.Component {
         }
 
         case "loadError":
-        case "updateError":
+        case "updateError": // TODO
```

Comment:
TODO what?

Reply:
Ahh that was just there for me. I already did what I wanted to do there, so removing that comment. thanks for noticing